var/spack/repos/builtin/packages/uchardet/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Uchardet(CMakePackage):
+    """uchardet is an encoding detector library, which takes a sequence of
+    bytes in an unknown character encoding without any additional information,
+    and attempts to determine the encoding of the text. Returned encoding names
+    are iconv-compatible."""
+
+    homepage = "https://www.freedesktop.org/wiki/Software/uchardet/"
+    url      = "https://github.com/BYVoid/uchardet/archive/v0.0.5.tar.gz"
```

Comment:
The README on Github says that the project moved to [freedesktop](https://cgit.freedesktop.org/uchardet/uchardet/).

Reply:
fixed it in https://github.com/spack/spack/pull/15937/commits/678779824efe97d52b33ad098b1c696c7acdbf68